### DISCUSSION TITLE
请告诉我们：你在用 Serverless Devs

### DISCUSSION THREAD

**Discussion by anycodes** [Serverless-Devs/Serverless-Devs#311] [Show and tell]
Hi，各位小伙伴，大家好哇。我们正在组织一个小的活动，叫《请告诉我们：你在用 Serverless Devs 》，欢迎各位小伙伴参加。参加方式在下面留言：

公司名称：   
公司规模：   
使用的业务和范围：   
公司Logo：   
公司地址：    
招聘信息：（可以带有联系人邮箱）    

对于提供完整信息的小伙伴并被认为是真实可靠信息的小伙伴，我们会将信息放在 Serverless Devs 的 repo 中作为展示。

**Comment by anycodes** [comment]
公司名称：阿里云
公司规模：10000+
使用的业务和范围：阿里云云原生Serverless团队使用该工具进行内部业务的开发、部署、运维、调试等。
公司Logo：
![image](https://user-images.githubusercontent.com/21079031/141402521-ed4b842c-6295-4126-ab28-2f845e5bf41a.png)
公司地址：浙江杭州
招聘信息：阿里云Serverless团队招有兴趣的小伙伴加入团队，前端、后端欢迎投递奖励，联系邮箱：jiangyu.ly@alibaba-inc.com

**Comment by king4sun** [comment]


**Reply by anycodes** [reply]
从整体上来说，Serverless Devs是可以解决Serverless应用全生命周期的管理，包括脚手架创建应用，应用开发，部署以及后期的运维、与CICD集成等相关能力。

**Comment by kekeee-shine** [comment]
公司名称：领悦数字
公司规模：10000+
使用的业务和范围：宝马集团数字化公司，目前很多项目正在使用serverless转型中
公司Logo：

**Reply by anycodes** [reply]
在使用Devs的过程中有任何问题，随时联系。也可以提Issue。

**Comment by comk88** [comment]
公司名称：single_user
公司规模：only_me
使用的业务和范围：in future 
公司Logo：pyman.com.cn
公司地址：pyman.com.cn

**Comment by pattazl** [comment]
公司名称：华勤技术
公司规模：10000+
使用的业务和范围：手机、平板、笔记本、服务器等消费类电子ODM/OEM厂商，目前部分数据项目开始尝试serverless
公司Logo：
公司地址：上海

**Comment by zmofei** [comment]
公司名称：Mofei Zhu (个人)
公司规模：1人
使用的业务和范围：host 个人博客
公司地址：zhuwenlong.com

**Comment by gudqs7** [comment]
公司名称：gudqs7 (个人)
公司规模：1人
使用的业务和范围：每天中午/晚上吃饭提醒 (cron 真好用)
公司地址：杭州

**Comment by zsqk-tech** [comment]
公司名称：掌上乾坤
公司规模：10+
使用的业务和范围：核心产品后端 (手机零售行业全链路一体化信息管理系统)
公司Logo：
![logo_zsqk](https://user-images.githubusercontent.com/12827907/203684665-cf451629-09ac-48cf-85ba-34edc9cfe685.png)

公司地址：山西晋城
招聘信息：欢迎喜欢小型技术团队的朋友加入我们 dev@zsqk.com.cn
